### ISSUE TITLE
RealTime Subscription is not working for 'All' SupabaseEventTypes.

### ISSUE THREAD

**Issue by krsikarya** [supabase/supabase-dart#32]
When given `SupabaseEventTypes.all` as SubscriptionType, it is not working. A week ago, it used to be work. 
 _realtimeSubscription =
        client.from(_table).on(SupabaseEventTypes.all, (payload) {
      switch (payload.eventType) {
      case 'INSERT':
      .......
      .........
      ............
}

BUT If `SupabaseEventTypes.insert` is given then RealTimeSubscription is working.
 _realtimeSubscription =
        client.from(_table).on(SupabaseEventTypes.insert, (payload) {
      switch (payload.eventType) {
        case 'INSERT':
      .......
      .........
      ............
}

Note: All dependencies are up-to-date.

I am not sure, what's wrong?

**Comment by rphly** [comment]
Experiencing this as well.

**Comment by dshukertjr** [comment]
Thank you @krsikarya and @rphly for reporting this!

I was able to confirm that this does happen with the Dart SDK but not the JavaScript SDK, thus it most likely is a bug on the Dart SDK. I will have a deeper look and let you know if I find something!

**Comment by dshukertjr** [comment]
Hi @krsikarya !

I found the cause in this library, and that issue seemed to have exited since the beginning of this library. When you said 
>A week ago, it used to be work.

was it with this library, or was it with any of the other libraries like the JavaScript library?

**Comment by dshukertjr** [comment]
Hey @krsikarya @rphly !

This issue should be fixed in the latest version of [supabase-dart](https://pub.dev/packages/supabase)! Thanks again for reporting this!

**Comment by w3b6x9** [comment]
@krsikarya @rphly just wanted to apologize for the lack of communication on changes to the `realtime` server (and `realtime-js`) which caused this issue. We recently merged in a change to the realtime server where we're no longer broadcasting `*` changes. Instead, `realtime-js` client would pick up `*` changes and broadcast appropriately (`INSERT`/`UPDATE`/`DELETE`). We merged in the `realtime-js` PR (see [here](https://github.com/supabase/realtime-js/pull/75)) back in February and waited a few months to merge in the `realtime` PR (see [here](https://github.com/supabase/realtime/pull/126)).

@dshukertjr thanks for the quick PR to fix this and @phamhieu thanks for the review!
